docs/topics/about-tools.adoc

```diff
@@ -1,14 +1,20 @@
 // Module included in the following assemblies:
 //
-// * docs/cli-guide/master.adoc
+// * docs/getting-started-guide/master.adoc
 
 [id="about-tools_{context}"]
 = About the tools
 
-The {ProductName} ({ProductShortName}) provides several tools to assist you in the various stages of your migration and modernization efforts. Review the details of each tool to determine which is right for your project.
+You can use any of these {ProductName} ({ProductShortName}) tools to help you migrate and modernize projects:
 
-* CLI
-* Web console
-* {PluginName} for Eclipse and Red Hat CodeReady Studio
-* {ProductShortName} extension for Visual Studio Code
-* {MavenName}
+* link:{ProductDocUserGuideURL}[CLI]: The {ProductShortName} command-line tool lets you assess and prioritize migration and modernization efforts for applications without the overhead of a graphical user interface. The {CLIName} includes customization options and integrates with external automation tools.
```

Comment:
Better but you could remove even more fluff. 

Example: "analyze/assess/prioritize ... applications" is used a lot. You could put that in the intro paragraph and then just include the differences for the CLI, IDE add-ons and console. 

Shorten the bullet text. No need to say twice : "Web console: The Web console. ..."

Why is Web capitalized? It's a generic word, so should be lower-case.

Reply:
You could rewrite the whole paragraph like this: "Customize the MTA options and automate the analyses and reports by using the MTA command line tool." (I don't think there is a hyphen here.)